### PR TITLE
yats:0.1.0

### PR DESCRIPTION
**Pull request by 0warning0error**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Serialize the Typst objects into bytes and deserialize the bytes into Typst objects, thus making it convenience to communicate between typst and wasm

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

**Comment by Enter-tainer** [comment]
Have you tried https://typst.app/docs/reference/data-loading/cbor/#definitions-encode ?

**Comment by 0warning0error** [comment]
No, I have not seen it. But it doesn't matter, because I just uploaded it for reference. JSON emerges despite the emergence of CBOR. Maybe this is in vain because I don't check it.
________________________________
发件人: ***@***.*** ***@***.***> 代表 Wenzhuo Liu ***@***.***>
发送时间: 2024年2月29日 18:13
收件人: typst/packages ***@***.***>
抄送: Zhao Yuanhang ***@***.***>; Author ***@***.***>
主题: Re: [typst/packages] serial:0.1.0 (PR #399)


Have you tried https://typst.app/docs/reference/data-loading/cbor/#definitions-encode ?

―

**Comment by Enivex** [comment]
There is https://typst.app/docs/reference/data-loading/json/#definitions-encode too

**Comment by laurmaedje** [comment]
As is the name is too obvious and canonical for what the package does. Please pick a more unique name. You can take a look at https://typst.app/docs/packages for inspiration.

**Comment by 0warning0error** [comment]
> As is the name is too obvious and canonical for what the package does. Please pick a more unique name. You can take a look at https://typst.app/docs/packages for inspiration.

Thanks. Give me time to rename my package.

File: packages/preview/yats/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Have a look at the example [here](./example.typ).
+
+## Usage
+
+Simply import 2 functions : `serialize`, `deserialize`.
+
+And enjoy it
+
+```typ
+#import "yats.typ": serialize,deserialize
```
Comment (laurmaedje) [inline packages/preview/yats/0.1.0/README.md:21]:
```suggestion
#import "@preview/yats:0.1.0": serialize, deserialize
```
it's better if the README (and ideally also the `example.typ`) shows an import from the package repository

Comment (0warning0error) [reply packages/preview/yats/0.1.0/README.md]: I updated both files. You can check.

**Comment by laurmaedje** [comment]
Sorry for the long wait, your latest changes slipped under my radar!

**Pull request merged**